src/scenes/PlantingSitesRouter/PlantingSitesCellRenderer.tsx

```diff
@@ -21,11 +21,13 @@ export default function PlantingSitesCellRenderer(props: RendererProps<TableRowT
   const { column, row, value, index } = props;
 
   const createLinkToPlantingSiteView = (iValue: React.ReactNode | unknown[]) => {
-    return (
-      <Link to={APP_PATHS.PLANTING_SITES_VIEW.replace(':plantingSiteId', row.id.toString())}>
-        {iValue as React.ReactNode}
-      </Link>
+    const isDraft = false; // TODO: lookup BE property when available
+    const to = (isDraft ? APP_PATHS.PLANTING_SITES_DRAFT_VIEW : APP_PATHS.PLANTING_SITES_VIEW).replace(
+      ':plantingSiteId',
+      row.id.toString()
```

Comment:
Sometimes prettier is kinda ugly 😓

Reply:
Yikes, that could have been just one line.

Reply:
If it is a long ternary I like this style personally
```
const to = (isDraft 
  ? APP_PATHS.PLANTING_SITES_DRAFT_VIEW
  : APP_PATHS.PLANTING_SITES_VIEW).replace(':plantingSiteId', row.id.toString());
```

Reply:
Same here, I have been using this pattern for ternaries that don't fit well on one line.